wpilibc/src/main/native/include/frc/AddressableLED.h

```diff
@@ -24,12 +24,27 @@ namespace frc {
  * By default, the timing supports WS2812B and WS2815 LEDs, but is configurable
  * using SetBitTiming()
  *
+ * Some LEDs use a different color order than the default RGB. The color order
+ * is configurable using SetColorOrder().
```

Comment:
It's a bit confusing to say here the default is RGB, but the ColorOrder enum says the default order is GRB.

Reply:
Whoops, that was leftover from when I thought the default was RGB